tensorrt_llm/recipes/db/tinyllama-test.yaml

```diff
@@ -0,0 +1,26 @@
+# TinyLlama 1.1B FP16 Recipe (Test Configuration)
+#
+
+scenario:
+  model: tinyllama
+  num_gpus: 1
+  target_isl: 1024
+  target_osl: 256
+  target_concurrency: 32
+  # Optional: Dataset generation parameters.
+  # This is useful for trtllm-bench to auto-generate dataset, so one can just specify this recipe
+  # to trtllm-bench without prior steps.
+  isl_stdev: 0       # Input sequence length standard deviation (0 = exact)
```

Comment:
Is this just for testing? I'm not sure we want this here -- are you thinking that the standard deviation would have a bearing on the recipe we choose?